### PR TITLE
Fix some typos and add some explanation for clarity

### PR DESCRIPTION
**Pull request by gaudenzkessler**

### CODE REVIEW

File: src/design.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ In the following we explain how Integritee works and how the different component
 ![vision](./fig/substraTEE-vision.png)
 *Integritee Target Architecture with Direct Invocation (future scenario)*
 
-* Shielding key: used by the integritee-client to encrypt the call in order to protect caller privacy. It is common to all enclaves.
+* Shielding key: used by the integritee-client to encrypt the call in order to protect caller privacy. It is common to all enclaves and created by *Distributed Key Generation* (DKG).
```
Comment (brenzi) [inline src/design.md]: this is actually not true in our current implementation. It is generated by the first enclave running and then shared with joining enclaves

File: src/remote_attestation.md [hunk 2]
```diff
@@ -1,57 +1,56 @@
...
+   1. accesses the correct STF state.
 
 ### Classical Remote Attestation Solution
 
-The standard use case for remote attestation involves a service provider (SP) like a video streaming service which wants to be sure his viewer application runs on a genuine SGX HW and respects DRM.
-The SP therefore requests a quote from the application and sends that quote to Intel Attestation Services (IAS) who sign off the quote if it is genuine.
+The standard use case for remote attestation (RA) involves a service provider (SP) like a video streaming service that wants to be sure that his viewer application runs on a genuine SGX HW and respects DRM.
+The SP therefore requests a quote from the application and sends that quote to the Intel Attestation Services (IAS) who signs off the quote if it is genuine. This process certifies the enclave to be genuine and the SP can safely run his calls. 
+
+However, that IAS only talks to registered clients. You need to register in order to get a SPID which needs to be supplied along with requests.
```
Comment (brenzi) [inline src/remote_attestation.md:19]: the service provider needs to register with Intel.
In the case above, the SP doesn't run the TEE hardware. In our case it does

File: src/fig/attestation.svg [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496.7000732421874 520.5999526977539" width="496.7000732421874" height="520.5999526977539">
```
Comment (brenzi) [inline src/fig/attestation.svg:1]: fix typo: certigfy

File: src/remote_attestation.md [hunk 4]
```diff
@@ -1,57 +1,56 @@
...
 
-The issue here is that IAS only talks to registered clients. You need to register in order to get a SPID which needs to be supplied along with requests.
+![Attestation of the enclave](./fig/attestation.svg)
 
 ### Attestation Registry On-Chain
 
-It isn't practical to ask every client to register with Intel and perform RA before every request. Therefore we'd rather let the integritee-worker operators attest their enclaves with IAS and write the signed quote and their certificate to the blockchain for everyone to verify.
+It isn't practical to ask every client to register with Intel and perform RA before every request. An integritee-worker operator attests his own enclave with IAS. The signed quote and the certificate is then written onto the blockchain and everyone can verify the genuineness of the enclave.
```
Comment (brenzi) [inline src/remote_attestation.md:25]: the certificate isn't just "written" to the blockchain! It is verified by the blockchain (the certificate is part of an extrinsic, but it never becomes part of the state)

File: src/remote_attestation.md [hunk 5]
```diff
@@ -1,57 +1,56 @@
...
-      * user data is hash of context:
-         * enclave-individual signing pubkey
-         * latest block hash
-   * ...
+  * MRENCLAVE (hash of enclave build)
+  * Product ID (hard-coded in the Integritee source)
+  * Security Version (hard-coded in Integritee source)
+  * user data is hash of context:
+     * enclave-individual signing pubkey
+     * latest block hash
```
Comment (brenzi) [inline src/remote_attestation.md]: don't do that. the report body is a hierarchical element of the type we describe here.

**Review by brenzi — CHANGES_REQUESTED** [review]

File: src/remote_attestation.md (hunk 2 above)
Comment (gaudenzkessler) [reply src/remote_attestation.md:19]: I only modified minor words in this paragraph.

File: src/remote_attestation.md (hunk 4 above)
Comment (gaudenzkessler) [reply src/remote_attestation.md:25]: I only modified minor words in this paragraph.

File: src/remote_attestation.md (hunk 5 above)
Comment (gaudenzkessler) [reply src/remote_attestation.md]: ok

**Pull request closed without merging**